frontend/components/ui/slideshow.tsx

```
@@ -0,0 +1,52 @@
+"use client";
+
+import { ChevronLeft, ChevronRight } from "lucide-react";
+import { useState } from "react";
+
+function Slideshow({ images, size }: { images: string[]; size: number }) {
```

ESLint

[error] 'size' is defined but never used.

(@typescript-eslint/no-unused-vars)